### PR TITLE
Add mypy types packages to dev-requirements

### PR DESCRIPTION
**Pull request by ManInFez**
Fixes failing mypy after version bump 0.901

### CODE REVIEW

File: .github/workflows/typing.yml [hunk 1]
```diff
@@ -22,10 +22,11 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install mypy==0.812
+        python -m pip install mypy
```
Comment (markusdregi) [inline .github/workflows/typing.yml:25]: It is a bit odd to add the type stubs to `dev-requirements`, but not `mypy` 🤔 Should we either add `mypy` as well, or create separate `type-requirements`?

**Review by markusdregi — COMMENTED** [review]
It is not "mypy types packages" but "type stub packages"... Not very important, but I suggest you update your commit message.

File: .github/workflows/typing.yml (hunk 1 above)
Comment (ManInFez) [reply .github/workflows/typing.yml:25]: Yes I agree, lets make a type-requirements file :+1:

**Review by markusdregi — APPROVED** [review]
Looking good 😎

**Pull request merged**